tools/PTE/marblescc_privInputs/marblescc-priv-chan2-constant-q-TLS.json

```diff
@@ -1,67 +0,0 @@
-{
-    "channelID": "_ch2",
-    "chaincodeID": "marbles_cc_priv",
-    "chaincodeVer": "v0",
-    "logLevel": "ERROR",
-    "invokeCheck": "TRUE",
-    "transMode": "Constant",
-    "transType": "Invoke",
-    "invokeType": "Query",
-    "targetPeers": "OrgAnchor",
-    "nProcPerOrg": "1",
-    "nRequest": "10",
-    "runDur": "1200",
-    "TLS": "enabled",
-    "channelOpt": {
-        "name": "testorgschannel2",
-        "action":  "create",
-        "orgName": [
-            "org3",
-            "org4"
-        ]
-    },
-    "burstOpt": {
-        "burstFreq": ["500", "200", "300", "100"],
-        "burstDur":  ["3000", "2000", "3000", "1000"]
-    },
-    "mixOpt": {
-        "mixQuery": "false",
-        "mixFreq": "3000"
-    },
-    "constantOpt": {
-        "recHist": "HIST",
-        "constFreq": "0",
-        "devFreq": "0"
-    },
-    "listOpt": {
-        "org1": ["peer1"],
-        "org2": ["peer1"]
-    },
-    "eventOpt": {
-        "timeout": "240000"
-    },
-    "ccType": "marblescc_priv",
-    "ccOpt": {
-        "keyIdx": [0],
-        "keyPayLoad": [2],
-        "keyStart": "0",
-        "payLoadMin": "1024",
-        "payLoadMax": "1024"
-    },
-    "deploy": {
-        "chaincodePath": "github.com/hyperledger/fabric-test/fabric-samples/chaincode/marbles02_private/go",
-        "fcn": "init",
-        "args": []
-    },
-    "invoke": {
-        "query": {
-            "fcn": "readMarble",
-            "args": ["marble"]
-        },
-        "move": {
-            "fcn": "initMarble",
-            "args": ["marble", "yellow","35","jerry"]
-        }
-    },
-    "ConnProfilePath": "ConnProfiles/test-network"
-}
```

Comment:
why doesn't git indicate this file is being deleted? It looks like you deleted all the lines in the file, but left the file there. Which is it?

Reply:
when I click "Files changed" button above it shows that this file is deleted. Not quite sure what you mean.